### ISSUE TITLE
Create cross-reference of obsolete to advanced format replacements

### ISSUE THREAD

**Issue by deoren** [rsyslog/rsyslog-doc#456]
This came up in a recent conversation with @rgerhards. As we go about pruning the older obsolete format content from the documentation it is important that we have a cross-reference in place to guide readers coming from older configurations to the equivalent parameters in the advanced format.

In some cases this is currently handled at the bottom of module docs, but since we're pruning that content as we go we'll need to consider a common index.

We may need to create a mock-up to get the process started just so we're on the same page.

**Comment by deoren** [comment]
related issue: #449

**Comment by deoren** [comment]
Note to self: grep rsyslog source for `eCmdHdlrGoneAway` as one way of helping to identify obsolete legacy directives that are no longer supported.

**Comment by rgerhards** [comment]
> Note to self: grep rsyslog source for eCmdHdlrGoneAway as one way of helping to identify obsolete legacy directives that are no longer supported.

yup - that definitely works (assuming I have all done cleanly... ;)).

**Comment by deoren** [comment]
Some notes from https://github.com/rsyslog/rsyslog-doc/issues/453#issuecomment-353718716:

> @rgerhards: I still think we should remove all obsolete format references and just put them into a single old-to-new mapping page. So it'll be removed from where it originally existed. On the new (single) page, we could add them at the bottom as "removed".
> 
> For new style/script parameters that disappear, I agree to keep them on the same page.

**Comment by deoren** [comment]
> @rgerhards: yup - that definitely works (assuming I have all done cleanly... ;)).

Awesome, I was hoping that would be a good indicator to work from.

**Comment by deoren** [comment]
Update: The work on PR #476 may resolve this in a slightly different way: mention the [`obsolete legacy`](http://www.rsyslog.com/doc/v8-stable/configuration/conf_formats.html) format directives in a small table right with the coverage for the [`advanced`](http://www.rsyslog.com/doc/v8-stable/configuration/conf_formats.html) format parameters.

This keeps the deprecated directive names right with the current parameters for easy reference.
